torchtext/datasets/babi.py

```diff
@@ -59,11 +58,11 @@ def __init__(self, path, text_field, only_supporting=False, **kwargs):
         fields = [('story', text_field), ('query', text_field), ('answer', text_field)]
         self.sort_key = lambda x: len(x.query)
 
-        with open(path, 'r', encoding="utf-8") as f:
+        with open(path, encoding="utf-8") as f:
```

Comment:
Is this relevant to the drop of python2 support?

Reply:
Yes, it's using Python 3 syntax that was not possible when also supporting Python 2.